README.md

```diff
@@ -200,6 +200,17 @@ await flatDirectory.download(key, {
 });
 ```
 
+Note: If you only need the download function, you can create a FlatDirector using the following pattern.
```

Comment:
what about the ethstorage obj?

Reply:
The ethstorage data key is bound to the wallet address, so a wallet is needed?

Reply:
only address is needed?

Reply:
fix